kano_updater/apt_wrapper.py

```diff
@@ -99,6 +100,51 @@ def refresh_instance(self):
 
         self._cache = apt.cache.Cache()
 
+    def _update_cache(self, progress, src_count, sources_list):
+        try:
+            self._do_update_cache(progress, src_count, sources_list)
+        except (
+                apt.cache.FetchFailedException, AptDownloadFailException
+        ) as err:
+            # Continue regardless, if it fails later because it can't grab a
+            # package then let it fail because of that; now that we've added
+            # in the development repositories it is more likely to happen.
```

Comment:
added in the dev repos?? What do you mean?

Reply:
I just moved the comment from before, I'll fix it up with an accurate comment.